broker/src/main/java/org/apache/rocketmq/broker/processor/AckMessageProcessor.java

```diff
@@ -209,7 +211,16 @@ private RemotingCommand processRequest(final Channel channel, RemotingCommand re
             && putMessageResult.getPutMessageStatus() != PutMessageStatus.SLAVE_NOT_AVAILABLE) {
             POP_LOGGER.error("put ack msg error:" + putMessageResult);
         }
+        decInFlightMessageNum(requestHeader);
         return response;
     }
 
+    private void decInFlightMessageNum(AckMessageRequestHeader requestHeader) {
```

Comment:
use the full name "decrement" maybe better

Reply:
done